olp-cpp-sdk-core/include/olp/core/geo/coordinates/GeoPoint.h

```diff
@@ -41,17 +41,37 @@ class CORE_API GeoPoint {
   GeoPoint(std::uint32_t xx, std::uint32_t yy);
 
   /**
-   * 2D absolute world coordinate.
+   * Absolute world X-coordinate.
    * Value range for each component is 32 bit unsigned integer
    */
   std::uint32_t x;
+  /**
+   * Absolute world Y-coordinate.
+   * Value range for each component is 32 bit unsigned integer
+   */
   std::uint32_t y;
-
+  /**
```

Comment:
Maybe we can use short-form `///` for the obvious methods (constr, destr, operators, etc.)
We dont need param and return.

Reply:
done